x/wasm/keeper/handler_plugin.go

```diff
@@ -22,12 +25,14 @@ type msgEncoder interface {
 
 // SDKMessageHandler can handles messages that can be encoded into sdk.Message types and routed.
 type SDKMessageHandler struct {
-	router   sdk.Router
-	encoders msgEncoder
+	router    sdk.Router
+	msgRouter *baseapp.MsgServiceRouter
```

Comment:
`msgRouter` is not included in `NewSDKMessageHandler`
https://github.com/CosmWasm/wasmd/blob/e85d769ee0db346e140e49b351c9878606667ab5/x/wasm/keeper/handler_plugin.go#L54